hazelcast/src/main/java/com/hazelcast/client/impl/spi/impl/listener/ClientListenerServiceImpl.java

```diff
@@ -153,12 +153,12 @@ public void provideStaticMetrics(MetricsRegistry registry) {
 
     @Probe(name = CLIENT_METRIC_LISTENER_SERVICE_EVENT_QUEUE_SIZE, level = MANDATORY)
     private int eventQueueSize() {
-        return eventExecutor.getWorkQueueSize();
+        return eventExecutor.getQueueSize();
     }
 
     @Probe(name = CLIENT_METRIC_LISTENER_SERVICE_EVENTS_PROCESSED, level = MANDATORY)
     private long eventsProcessed() {
-        return eventExecutor.processedCount();
+        return eventExecutor.getCompletedTaskCount();
```

Comment:
nit: I suggest you to not use "get" prefix, if field doesn't have setter. Just `eventExecutor.completedTaskCount()`

Reply:
I see it comes from `ManagedExecutorService`, closing my answer.